backend/app/routes/admin/getAdmins.js

```diff
@@ -41,6 +41,9 @@ module.exports = async (req, res, next) => {
     });
     return next(error);
   }
-  res.status(200).send(response);
+  // I have changed the response to an object instead of an array
+  const obj = Object.assign({}, ...response);

```

Comment:
How is this solving the issue?

Reply:
I had asked about this before. This  code is sending the response as an object just as you had asked. I have attached the screenshot above. But if it's not okay please tell the required changes..

Reply:
Yes but that code was already there. We need to add the feature of sending the details of logged in user only. Can you help me understand how is your PR solving that?

Reply:
Here is what i did. Using the dummy credentials, I found out the token for the dummy admin using the login route. Then I   sent a get request to admin/  using the token which displayed the details of the dummy admin. Should I make a separate route to check that user is logged in and then display the details?

Reply:
Please provide api testing files for the same

Reply:
Your code doesn''t work. It doesn't return the details of logged in user always. Also it has multiple bugs and throws error

Reply:
No, you don't require a new route, you need to use the same route